proto/topodata.proto

```diff
@@ -147,6 +158,12 @@ message Tablet {
   // for legacy behavior.
   int32 mysql_port = 13;
 
+  // Timestamp of backup that this tablet was restored from
```

Comment:
Do you have an example use case that requires these fields? I general, we're trying to use topo for fewer and fewer things over time. I'd only want to add new fields if they're really necessary. If it's not necessary now, but we feel it might be later, let's add it later when we're sure.

Reply:
Yes we do. Once the recovery keyspace has been restored, you might want to replay binlogs to bring it up to the time you really want to get to. backup_time and replication_position are there to allow you to pin a starting point for the binlog replay.

Reply:
I can see why you'd need the replication position for that, but isn't mysqld the source of truth for "what position is this tablet at now?"

Also, what would you use the backup time for? If we want to hit a particular point in time, we'll need to have some mechanism for figuring out what replication position we should set as our goal within the binlogs. If we already know the goal replication position, what do we need the real-time timestamp for?

Lastly, the type of use case I meant is one that specifically requires that these fields live in topology rather than somewhere else. Knowing that we'll need this information to be available somewhere is not quite enough to justify adding these fields. We also need justification that topology is the best place for this information.

Reply:
This has been removed from Tablet proto. We save in local_metadata instead.